test/other/embind_tsgen.cpp

```diff
@@ -159,6 +160,6 @@ int Test::static_property = 42;
 
 int main() {
   // Main should not be run during TypeScript generation.
```

Comment:
Does this comment need updating too?   Why not continue to abort here?

Reply:
Oh I see you are setting that it can actually be run.. maybe the comment needs an update though?